src/routes/volume/VolumeBulkActions.js

```diff
@@ -214,6 +218,7 @@ function bulkActions({
     { key: 'updateReplicaDiskSoftAntiAffinity', name: 'Update Replica Disk Soft Anti Affinity', disabled() { return selectedRows.length === 0 } },
     { key: 'trimFilesystem', name: 'Trim Filesystem', disabled() { return selectedRows.length === 0 || notAttached() } },
     { key: 'updateFreezeFilesystemForSnapshot', name: 'Update Freeze Filesystem For Snapshot', disabled() { return selectedRows.length === 0 } },
+    { key: 'offlineRebuild', name: 'Offline Replica Rebuild', disabled() { return selectedRows.length === 0 } },
```

Comment:
If any selected volume is in the `faulted` state, the action will still be available. The backend handles it properly, so no error will be shown even if the action is skipped for faulted volumes.